visualization/app/material/matMenu.scss

```diff
@@ -0,0 +1,16 @@
+.mat-menu-content {
+	.menu-header {
+		font-size: 9pt;
+		height: 20px;
+		line-height: 20px;
+	}
+
+	// overwrite global app.scss style
+	button {
+		margin: unset;
+	}
+}
+
+.mat-menu-item {
```

Comment:
.mat-menu-item is a child of .mat-menu-content and could be in the same paranthesis level as .menu-header above

Reply:
done